src/QsCompiler/CommandLineTool/Commands/Build.cs

```diff
@@ -166,6 +166,30 @@ private static BuildOptions FromResponseFiles(IEnumerable<string> responseFiles)
                 });
         }
 
+        /// <summary>
+        /// Publishes performance tracking data.
+        /// Logs errors if something went wrong while tracking performance or publishing results.
+        /// </summary>
+        private static void PublishPerformanceTrackingData(string perfFolder, ConsoleLogger logger)
```

Comment:
Does the performance logging feature have any tests?

Reply:
It currently does not have any tests. I changed the target branch of this PR from `master` to `feature/perf-tracking-additions`. I can then add tests as a separate PR that goes into that branch, and once both are merged we can merge the feature branch into `master`.